eng/publish/PublishVSCodeExtensionUsingMI.ps1

```diff
@@ -66,7 +66,7 @@ try {
                 if ($nugetPackageName -like '*.symbols.nupkg') {
                     Write-Host "Skipping publishing symbol package $nugetPackagePath"
                     # Use 'continue' to skip to the next iteration
-                    continue
+                    return
```

Comment:
Also are you sure you are not terminating all? return here will exit the publish script. At the first symbol package this will stop

Reply:
actually that is the opposite. Return will continue to the next item, and Continue will exit the loop

Reply:
???

Reply:
because this is more verbose. Here is a sample log

https://dev.azure.com/dnceng/internal/_build/results?buildId=2570763&view=logs&j=8b2f3529-484e-5976-a626-d4a8d47f3428&t=18943edd-a66d-52d4-6098-50cd321688d9

Reply:
why is making filter correct more verbose?